### ISSUE TITLE
Enable HybridRenderer_v2

### ISSUE THREAD

**Issue by zrussell64** [dokidzung/32BitJam#40]
Need HybridRenderer to use the DOTS/ECS.
Follow this guide: https://docs.unity3d.com/Packages/com.unity.rendering.hybrid@0.11/manual/creating-a-new-hybrid-renderer-project.html#hybrid-renderer-v2

**Comment by zrussell64** [comment]
The project doesn't allow the use of v2, but we can still use v1. For the scope of this project, I think we will be fine with that version.

**Comment by charlesamacon** [comment]
Yeah, it looks like v2 is only supported by the HDRP, which I don't think we're using. I vote to close this issue, at least for now
